testutil/integration/app.go

```diff
@@ -659,6 +755,11 @@ func (app *App) RunMsg(t *testing.T, msg sdk.Msg, option ...RunOption) *codectyp
 	if cfg.AutomaticCommit {
 		defer func() {
 			_, err := app.Commit()
+			if cfg.ErrorAssertion != nil && err != nil {
+				cfg.ErrorAssertion(err)
```

Comment:
Should we log something when there's an error but `cfg.ErrorAssertion` is nil?

Reply:
I don't think it's necessary and I don't think we have the scope here to know whether it's an issue. If `cfg.ErrorAssertion` is set then it should be responsible for determining whether a nil error is acceptable.